test-yolo.py

```diff
@@ -0,0 +1,262 @@
+#!/usr/bin/env python
+# coding=utf-8
+
+#
+# Usage:
+#
+# Run all tests against PHP 7.3:
+#
+#   ./test-yolo.py ci-drupal-8
+
+
+import os
+import pwd
+import errno
+import pprint
+import re
+import subprocess
+import random
+import signal
+import sys
+import time
+import json
+
+import requests
+import unittest
+import shlex
+
+# pass in site name as an argument to specify a Drupal 7 or Drupal 8 site
+
+siteName = sys.argv[1]
+del sys.argv[1]
+
+commandPrefix = "terminus drush "
+
+#drupal7Site = "ci-drupal-7"
+#drupal8Site = "ci-drupal-8"
+
+class EnvCacheClearTestCase(unittest.TestCase):
+    def fixtureCommand(self):
```

Comment:
This only runs env:clear-cache; maybe the method name should reflect that.

Reply:
Ah, I see; every class has its own fixture command, following the pattern of the test-yolo script in rpmbuild-php.

Reply:
In rpmbuild-php, the URL points to a fixture page, ergo fixture URL (the url of the fixture). In this case, what is being returned is a command string; the fixture is the site the command operates on. The name is therefore not quite correct.

Reply:
I will rename it to command() and getCommandResult(), let me know if you have a different preferred naming

Reply:
Or if you think it is better to refactor the code and/or put everything in one function, let me know!